src/components/MessageList/hooks/useEnrichedMessages.ts

```diff
@@ -72,10 +79,11 @@ export const useEnrichedMessages = <
     messagesWithDates = insertIntro(messagesWithDates, headerPosition);
   }
 
+  const groupStylesFn = groupStyles ? groupStyles : getGroupStyles;
```

Comment:
Change this to: `const groupStylesFn = groupStyles || getGroupStyles;` to match other overrides in the library.